client/lib/plans/features-list.js

```diff
@@ -408,7 +408,7 @@ export const FEATURES_LIST = {
 	[ constants.FEATURE_VIDEO_UPLOADS_JETPACK_PRO ]: {
 		getSlug: () => constants.FEATURE_VIDEO_UPLOADS_JETPACK_PRO,
 		getTitle: () =>
-			i18n.translate( '{{strong}}Unlimited{{/strong}} Video Hosting', {
+			i18n.translate( '{{strong}}Unlimited{{/strong}} Video hosting', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.